packages/app/app/components/LibraryView/LibraryFolderTree/styles.scss

```diff
@@ -0,0 +1,8 @@
+$header-background-color: transparent;
+$background-color: transparent;
+//$table-prefix: BaseTable !default;
+
+$border: none;
+$row-hovered-background-color: hsla(0, 0%, 100%, .2);
+
+@import '~react-base-table/es/_BaseTable.scss';
```

Comment:
Can this be moved to the top of the file?

Reply:
I'm not sure of the details of how scss variable-insertion works, but I'll try it.

Reply:
Okay yeah, I tried placing the import at the top, and it makes it ignore the variable assignments. (the variables have to be set before the import, since the imported scss makes use of those variables, and the variable assignments and reads apparently "run" sequentially)